.github/workflows/test.yml

```
@@ -59,6 +59,8 @@ jobs:
 
       - name: Install
         run: npm ci
+          # Increase GitHub API limit.
+          GITHUB_TOKEN: ${{ github.token }}
```

This should be under an `env` key I guess?
```suggestion
        env:
          # Increase GitHub API limit.
          GITHUB_TOKEN: ${{ github.token }}
```